easybuild/easyconfigs/s/snappy/snappy-1.1.9_use-default-rtti.patch

```diff
@@ -0,0 +1,27 @@
+Disable any modification of the RTTI setting and follow what is set in the environment.
+By default it is enabled in GCC and software might rely on the RTTI symbols in snappy.
```

Comment:
I'm guessing that 'software' was dm-reverb? https://github.com/easybuilders/easybuild-easyconfigs/pull/15104
It might be useful to add that explicitely: "... and software (e.g. dm-reverb) might rely ...". That way it's clear that this is not a purely theoretical case, but we've hit it in practice.

Reply:
Indeed, I did hit this issue with dm-reverb. But I prefer to keep the description of the patch as is to avoid confusing other users in thinking that this fix is software specific. Disabling RTTI will be disruptive in general until all software downstream updates to not rely on `typeinfo` symbols. So we can either ignore this version of snappy until everybody updates or we can roll-back this change.

Rolling back to have RTTI enabled is very safe as it just adds extra symbols, so software programmed for `-fno-rtti` will still work, and it is also the default behaviour of GCC. 

I will expand the description of this PR in case somebody wants to track this deeper.

Reply:
> But I prefer to keep the description of the patch as is to avoid confusing other users in thinking that this fix is software specific. 

That's also true. Well, I don't really have strong feelings about it either way, it was just a thought. 

>Disabling RTTI will be disruptive in general until all software downstream updates to not rely on typeinfo symbols. So we can either ignore this version of snappy until everybody updates or we can roll-back this change.

Makes sense, and sounds like the safe approach indeed. Since you mentioned you wanted to expand the description a bit, I'll not merge it yet. Just give me a shout when you're done and I'll do the merge :)

Reply:
@casparvl I already expanded the description of this PR. If you would like to add anything else let me know 🙂

Reply:
Sorry, I misread: I thought you wanted to still change something in the patch description, but you meant the PR description. All good now, I'll merge it :)